### PR TITLE
Rework routing, controllers, remove caching. 

### PR DESCRIPTION
**Pull request by bsord**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
     "react-dom": "^16.8.6",
     "react-router-dom": "^5.0.0",
     "react-scripts": "^3.0.1",
-    "socket.io-client": "^2.2.0"
+    "socket.io-client": "^2.2.0",
+    "yarn": "^1.16.0"
```
Comment (jonfairbanks) [inline client/package.json]: Is this required?

Comment (bsord) [reply client/package.json]: No. Not sure how that happened.

File: server/package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "socket.io": "^2.2.0",
     "util": "^0.11.1",
     "valid-url": "^1.0.9",
-    "winston": "^3.2.1"
+    "winston": "^3.2.1",
+    "yarn": "^1.16.0"
```
Comment (jonfairbanks) [inline server/package.json]: Similar here. Needed?

Comment (bsord) [reply server/package.json]: No, disregard. Dev env config issues.

File: yo.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    env:
+      PORT: 4095
+    watch:  true
+
+  - name:   yo-server
+    script: index.js
+    cwd:    server
+    env:
+      PORT: 4090
+    watch:  true
```
Comment (jonfairbanks) [inline yo.yaml]: This looks like your personal yaml?

**Comment by jonfairbanks** [comment]
Gracias

**Pull request merged**